### PR TITLE
feat(trees-map): add trees map page, add Mapbox wrapper component 

### PR DESCRIPTION
**Pull request by yossi-eynav**

### CODE REVIEW

File: client/src/components/Map/map.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import Mapbox from 'react-map-gl';
+import PropTypes from 'prop-types';
+
+mapboxgl.workerClass = MapboxWorker;
+
+
+export const Map = ({ children, onClick, initialViewState, interactiveLayerIds, style }) => {
+	return (
+		<Mapbox
+			mapboxAccessToken="pk.eyJ1IjoieW9zc2ktZXluYXYiLCJhIjoiY2xjcDgwcDJpMXJldTNybXFzaXNwd2FpYSJ9.JQLLuh4VYjrxgpxWIrYsGA"
```
Comment (yossi-eynav) [inline client/src/components/Map/map.jsx:18]: should be extracted outside (no security issue since it will be used in client side anyway)

Comment (GalGend) [reply client/src/components/Map/map.jsx:18]: we have client side config, can be added there

File: client/src/pages/TreeMap/index.jsx [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+
+	return (
+		<Typography component="p" color="textPrimary"> <strong>כתובת: </strong>{address}</Typography>
+	);
+};
+
+function ContentField(props) {
+	// eslint-disable-next-line react/prop-types
+	const { field, fieldBold } = props;
+	const text = field || 'לא מצוין';
```
Comment (GalGend) [inline client/src/pages/TreeMap/index.jsx:150]: please use translation hook + add to HE_IL file

File: client/src/pages/TreeMap/index.jsx [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+								<div className="map-title">
+									{place && <span className="btn btn-light disabled">{place}</span>}
+									{timeToObjection && <span className="btn btn-light map-title-left">{timeToObjection}</span>}
+								</div>
+								<Typography
+									gutterBottom
+									variant="h5"
+									component="h2"
+									color="textPrimary"
+								>
```
Comment (GalGend) [inline client/src/pages/TreeMap/index.jsx:88]: use translation file

File: client/src/pages/TreeMap/index.jsx [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+					longitude:  34.82341235969558,
+					latitude: 31.812881942711954,
+					zoom: 7
+				}}>
+				<MapImage/>
+				{ geojson &&  <Source id="trees-data" type="geojson" data={geojson}>
+					<Layer {...polygonStyle} />
+					<Layer  id='high-tree' {...symbolStyle({ image: 'large-tree-pin', size: 0.2 })}  filter={['>', ['get', 'total_trees'], 50]}/>
+					<Layer  id='low-tree' {...symbolStyle({ image: 'medium-tree-pin', size: 0.3 })}  minzoom={12} filter={['<', ['get', 'total_trees'], 10]}/>
+					<Layer  id='mid-tree' {...symbolStyle({ image: 'medium-tree-pin', size: 0.3 })}  minzoom={10} filter={['all', ['>=', ['get', 'total_trees'], 10], ['<=', ['get', 'total_trees'], 49]]}/>
```
Comment (GalGend) [inline client/src/pages/TreeMap/index.jsx:69]: Please take the design with Dror

File: client/src/pages/TreeMap/index.jsx [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+			properties: location && location.properties
+		});
+	}, []);
+
+	const selectedLocationProps = (locationInfo && locationInfo.properties) || '';
+
+	const timeToObjection = timeToObjectionText(selectedLocationProps.last_date_to_objection);
+	const place = selectedLocationProps.place;
+
+	if (isMobile()) {return null;}
```
Comment (gruppin) [inline client/src/pages/TreeMap/index.jsx:52]: should remain null IMO. if we don't know what to do here, at least put an image / message to the user that this feature is available only in desktop.

**Comment by yossi-eynav** [comment]
closed this branch, created a new one.
will address the comments there 👍

File: client/src/pages/TreeMap/index.jsx (hunk 2 above)
Comment (yossi-eynav) [reply client/src/pages/TreeMap/index.jsx:150]: I copy-pasted it 😱

**Pull request closed without merging**